setup.py

```diff
@@ -7,10 +7,8 @@
 # ----------------------------------------------------------------------------
 
 from setuptools import find_packages, setup
-
```

Comment:
**Lint:**
These lines were separating the imports intentionally, as the above group are effectively stdlib, whereas the bottom is not (although is vendored).

To undo those changes easily with git, you can use
```
git checkout master setup.py
```
while in your `trimming` branch and then commit the results :)